src/invidious/routes/errors.cr

```diff
@@ -4,15 +4,17 @@ module Invidious::Routes::ErrorRoutes
     if HOST_URL.empty? && env.request.path.starts_with?("/v1/storyboards/sb")
       return env.redirect "#{env.request.path[15..]}?#{env.params.query}"
     end
-
+    
     if md = env.request.path.match(/^\/(?<id>([a-zA-Z0-9_-]{11})|(\w+))$/)
       item = md["id"]
 
       # Check if item is branding URL e.g. https://youtube.com/gaming
-      response = YT_POOL.client &.get("/#{item}")
-
+      headers = HTTP::Headers{
+        "Cookie" => "SOCS=CAE" # Cookies to prevent redirects to Cookie Consent Page CAE~Reject all, CAA~showing the cookie banner, CAI~Accept all
+      }
```

Comment:
A small change for legibility:
```suggestion
      # Cookie to prevent redirects to the cookie consent page.
      # Cookie values: CAE = Reject all, CAA = show the cookie banner, CAI = Accept all
      headers = HTTP::Headers{"Cookie" => "SOCS=CAE"}

```